.github/workflows/trigger-dynamic-readme-update.yaml

```diff
@@ -14,7 +14,51 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        repository: [thoughtbot/high_voltage]
+        repository:
```

Comment:
So many of these I have never heard of 😅

Reply:
While I'm usually in the camp of "nothing is impossible", I think the GHAs are not really _that_ smart as to infer which external repo's workflow you want to trigger.

Reply:
Same! :D

> While I'm usually in the camp of "nothing is impossible", I think the GHAs are not really that smart as to infer which external repo's workflow you want to trigger.

yeah, that makes sense. Unless in the repo that has the `workflow_dispatch` key, we could add which workflow path to be listening to updates. So the triggering would work the other way around. I wonder if that's something we could pitch as an idea?